packages/block-library/src/heading/autogenerate-anchors.js

```diff
@@ -32,11 +32,14 @@ const getTextWithoutMarkup = ( text ) => {
  */
 const getSlug = ( content ) => {
 	// Get the slug.
-	return trim(
-		deburr( getTextWithoutMarkup( content ) )
+	return (
```

Comment:
@tyxla I'm not sure here, but should we standardize the usage of this? 

In the other spot (#41687) where we added the `removeAccents` we added the following transformations: 

https://github.com/WordPress/gutenberg/blob/6bf86eb3eb960fcceeb8e3b09d280449c45c1fdf/packages/url/src/clean-for-slug.js#L27-L35


Just want to make sure we don't have unexpected results anywhere :)

Reply:
Hm, thanks for double-checking @fabiankaegy, but that's not entirely true 😉  If you take a look at the [commit](https://github.com/WordPress/gutenberg/commit/1a9b1d0b98f5c555a77c1eb5bba0ccce14457100) you'll notice that `.replace( /[\s\./]+/g, '-' )` and `.replace( /[^\p{L}\p{N}_-]+/gu, '' )` were already there - we just added comments for them.  The rest - the `.replace( /(^-+)|(-+$)/g, '' )` essentially, we added in both places - here and in #41687. So whatever we added there, we're adding it here. 

In terms of standartizing, I'm not really sure why the first `replace` is missing here, but perhaps there's a good reason for that? In any case, this isn't within the goal of this PR, so if we decide we need to have the first `replace` added here, I'm happy to follow up in another PR.

Reply:
Yeah that makes perfect sense. 

And yeah I was badly worded on my end :) my regards was about the fact that the two spots where we try to remove all the accents do it differently. Which was true even before the pr :) and yes we should not combine it with the refactor of removing lodash

Reply:
To be honest, I'm not even sure if the first `replace` is necessary. By replacing all non-alphanumeric characters with a dash (which is different from what we do in the other PR - just removing them), we achieve the same as in the other PR. So we can as well do the same in the other place, which would definitely help standardized the practices 😉 So you definitely bring a good point there.